go.mod

```diff
@@ -5,7 +5,7 @@ go 1.22.5
 require (
 	github.com/deckarep/golang-set/v2 v2.6.0
 	github.com/smartcontractkit/chain-selectors v1.0.23
-	github.com/smartcontractkit/chainlink-common v0.2.3-0.20241001210038-dd59341432bd
+	github.com/smartcontractkit/chainlink-common v0.2.3-0.20241002211418-707d968168d4
```

Comment:
What was the common bump? Unrelated?

Reply:
Was debugging locally, this is latest cl-common so it should be okay to leave it as is.